### PR TITLE
Adding CONTRIBUTING.md and updated the content

### PR DESCRIPTION
**Pull request by Supun94**
PR for issue #7

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* Do not open an issue on GitHub until you have collected positive feedback about the change. GitHub issues are primarily intended for bug reports and fixes.
+
+
+#### **Do you have questions about the source code?**
+
+
+</br>
+
+
+RetroAchievements.org is a volunteer effort. We encourage you to pitch in and [join the team](https://github.com/joestrong/retroachievements-composer/graphs/contributors)!
```
Comment (joestrong) [inline CONTRIBUTING.md]: This is not for the RetroAchievements.org site itself, so can you change 'RetroAchievements.org' for 'This project'

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+Thanks! :heart: :heart: :heart:
+
+
+RetroAchievements.org Team
```
Comment (joestrong) [inline CONTRIBUTING.md]: Can you remove this line (I'm not part of the retro achievements site itself)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+CONTRIBUTING file inspired/based on [rails contrubiting](https://github.com/rails/rails/blob/master/CONTRIBUTING.md)
```
Comment (joestrong) [inline CONTRIBUTING.md]: Can you remove this line?

**Review by joestrong — CHANGES_REQUESTED** [review]
Really nice, thanks! Just a couple of amends please.

Also, can you change this to branch off the latest version of the develop branch (like the new contributing guide says 😄 )

**Comment by Supun94** [comment]
@joestrong I have done the changes as you mentioned. Please review :) . Do you need this PR for [develop branch](https://github.com/joestrong/retroachievements-composer/tree/develop) ? (Sorry I didn't what you mentioned in the comment)

**Comment by joestrong** [comment]
Yeah, all PRs need to be a branch from develop, with the intent to merge into develop.

Can you rebase your commits onto a branch, branching off the tip of the latest develop branch? And rename the branch to something more fitting to the changes

**Comment by Supun94** [comment]
@joestrong Please find this #11 PR and review.

**Pull request closed without merging**